### PR TITLE
feat: new material comparison script

### PR DESCRIPTION
**Pull request by asalzburger**
This PR overwrites the material comparison script, that was based on ROOT with an uproot version.

As one can see here: the Gen2 navigation still needs debugging!

MaterialComparison

### CODE REVIEW

**Comment by asalzburger** [comment]
@benjaminhuth - be critical and teach me better python !

File: Examples/Scripts/MaterialMapping/material_comparison.py [hunk 1]
```diff
@@ -1,239 +1,139 @@
...
+y_ratio_values = []
+y_ratio_errors = [0.0 for i in range(x_bins)]
+
+for ifile, (fname, label, color, mfc, cmap, msize) in enumerate(files):
+
+    # labels += [ label+" mean" ]
+    tree = uproot.open(fname)
+
+    # fig, ax = plt.subplots()
+    y_arr = np.array(tree[y_val_name].array())
```
Comment (benjaminhuth) [inline Examples/Scripts/MaterialMapping/material_comparison.py:53]:
```suggestion
    y_arr = tree[y_val_name].array(library="np")
```
I think that should work as well, similar for the other cases

File: Examples/Scripts/MaterialMapping/material_comparison.py [hunk 2]
```diff
@@ -1,239 +1,139 @@
...
-def TH1D_from_TProf(tprof):
-    h = TH1D(
-        tprof.GetName() + "_th1",
-        tprof.GetTitle(),
-        tprof.GetNbinsX(),
-        tprof.GetXaxis().GetXmin(),
-        tprof.GetXaxis().GetXmax(),
+# Make a file list, e.g.
+files = [
+    (
```
Comment (benjaminhuth) [inline Examples/Scripts/MaterialMapping/material_comparison.py:8]: I think maintainability could be improved using `namedtuple`s

File: Examples/Scripts/MaterialMapping/material_comparison.py [hunk 3]
```diff
@@ -1,239 +1,139 @@
...
+        "none",
+        "Greens",
+        5,
+    ),
+]
+
+fig, axs = plt.subplots(2, 1, sharex=True)
+fig.subplots_adjust(hspace=0.05)
+
+x_range = (-4.0, 4.0)  # [-math.pi, math.pi ] #
```
Comment (benjaminhuth) [inline Examples/Scripts/MaterialMapping/material_comparison.py:37]: should these comments stay?

**Review by benjaminhuth — APPROVED** [review]
Not a lot to critisize from my perspective - looks like proper python code to me! I only have a few minor comments.

File: Examples/Scripts/MaterialMapping/material_comparison.py (hunk 1 above)
Comment (asalzburger) [reply Examples/Scripts/MaterialMapping/material_comparison.py:53]: Ok, thanks, will change.

**Comment by asalzburger** [comment]
Hey @benjaminhuth - can you have a second look?

**Review by benjaminhuth — APPROVED** [review]
Looks good!

**Comment by andiwand** [comment]
removing `automerge` so we can cut v36

**Pull request merged**